recipes/ktx/all/conanfile.py

```diff
@@ -116,8 +116,8 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "Ktx")
-        self.cpp_info.set_property("cmake_target_name", "KTX")
-        self.cpp_info.components["libktx"].set_property("cmake_target_name", "ktx")
+        self.cpp_info.set_property("cmake_target_name", "KTX::KTX")
+        self.cpp_info.components["libktx"].set_property("cmake_target_name", "KTX::ktx")
```

Comment:
```suggestion
        self.cpp_info.set_property("cmake_target_name", "KTX::ktx")
```

Reply:
Maybe for this one we want to leave the component workaround and add a #FIXME to avoid duplicating setting the defines, system_libs and so on to the root cpp_info?

Reply:
@SpaceIm as @czoido mentioned, if I remove it, I will have to add all the attributes to the root cpp_info, right? So, just for me to know and for future PR, should we wait for another iteration? or better to have it already done?